[2022-04-11 13:27:07,372] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: download_data_dag.download_data scheduled__2019-01-01T00:00:00+00:00 [queued]>
[2022-04-11 13:27:07,385] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: download_data_dag.download_data scheduled__2019-01-01T00:00:00+00:00 [queued]>
[2022-04-11 13:27:07,385] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-04-11 13:27:07,385] {taskinstance.py:1239} INFO - Starting attempt 4 of 4
[2022-04-11 13:27:07,385] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-04-11 13:27:07,399] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): download_data> on 2019-01-01 00:00:00+00:00
[2022-04-11 13:27:07,409] {standard_task_runner.py:52} INFO - Started process 323690 to run task
[2022-04-11 13:27:07,416] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'download_data_dag', 'download_data', 'scheduled__2019-01-01T00:00:00+00:00', '--job-id', '1666', '--raw', '--subdir', 'DAGS_FOLDER/nfl_to_gcs.py', '--cfg-path', '/tmp/tmpxt07906f', '--error-file', '/tmp/tmpz4zgvhoe']
[2022-04-11 13:27:07,417] {standard_task_runner.py:77} INFO - Job 1666: Subtask download_data
[2022-04-11 13:27:07,489] {logging_mixin.py:109} INFO - Running <TaskInstance: download_data_dag.download_data scheduled__2019-01-01T00:00:00+00:00 [running]> on host 4ed3eda454fd
[2022-04-11 13:27:07,557] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=download_data_dag
AIRFLOW_CTX_TASK_ID=download_data
AIRFLOW_CTX_EXECUTION_DATE=2019-01-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-01-01T00:00:00+00:00
[2022-04-11 13:27:07,872] {python.py:175} INFO - Done. Returned value was: None
[2022-04-11 13:27:07,888] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=download_data_dag, task_id=download_data, execution_date=20190101T000000, start_date=20220411T132707, end_date=20220411T132707
[2022-04-11 13:27:07,950] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-11 13:27:07,998] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
